### ISSUE TITLE
Deconvolution runtime

### ISSUE THREAD

**Issue by haoala** [deconvolutionpaper/deconvolution#6]
Thanks for this paper, I really enjoyed reading it.

I replaced all the batch norm layers in a ResNeXt-50 model with `ChannelDeconv(block=64)` layers, but I found that training takes much longer doing so, running about 30% slower. Did you notice this too with your experiments? Do you have any suggestions for speeding up the deconvolution layers?

**Comment by yechengxi** [comment]
At this time there can be significant overhead because the layers have not been optimized by the deep learning platforms yet. Simply calling the im2col function without doing anything can take longer than the convolution layer. We have to tolerate it before it is integrated into pytorch.

**Comment by PkuRainBow** [comment]
@yechengxi According to your segmentation experiments, we also find that the overall runtime of ResNet-101 FCN w/ Deconvolution is nearly 2x compared with the runtime ResNet-101 FCN w/ SyncBN. 

Hoping to see the optimized version soon.
